src/services/container/ContainerService.js

```diff
@@ -18,134 +19,179 @@ export class ContainerService extends BaseService{
     /**
      * Creates a new container and returns the container object with an id
      * @param {string} name name of the new container
-     * @param {string} householdId id of the household where the container is held
      * @returns the container object that was created
+     * @throws errors if a container was not able to be made in DB
      */
-    async createContainer(name, householdId, icon = this.defaultIcon){
-        this.__UseCollection(this.CONTAINER_COLLECTION)
-        
-        
-        const container = new Container(name, householdId, [], icon)
-
-        const newDocId = await this.__CreateEntity( container.toDocument() )
-        container.id = newDocId
-        return container
+    async createContainer(name){
+        try{
+            this.__UseCollection(this.CONTAINER_COLLECTION)
+            
+            const defaultIcon = {
+                name: "DefaultName",
+                color: "DefaultColor",
+                type: "DefaultType"
+            }
+            const container = new Container(name, [], defaultIcon)
+
+            const newDocId = await this.__CreateEntity( container.toDocument() )
+            container.id = newDocId
+            return container
+        }
+        catch(err){
+            logError(err)
+            throw err
+        }
     }
 
     /**
      * Creates a new container and returns the container object with an id
      * @param {string} name name of the new container
-     * @param {string} householdId id of the household where the container is held
      * @param {Object} icon object with attributes name, color, type as strings
-     * @returns the container object that was created, or null if the icon was invalid
+     * @returns the container object that was created
+     * @throws errors if icon is invalid or if container was not able to be created in DB
      */
-    async createContainerWithIcon(name, householdId, icon){
-        this.__UseCollection(this.CONTAINER_COLLECTION)
+    async createContainerWithIcon(name, icon){
```

Comment:
This does not need to be a separate function. The defaultIcon argument you see in red on previous line 24 would solve this issue